### PR TITLE
Feature/dp 4947 kaart react draw tool tweaks

### PR DESCRIPTION
**Pull request by RRMoelker**

### CODE REVIEW

File: src/map/components/points-available/PointsAvailable.test.jsx [hunk 1]
```diff
@@ -2,34 +2,32 @@ import React from 'react';
 import { shallow } from 'enzyme';
```
Comment (jasperswart) [inline src/map/components/points-available/PointsAvailable.test.jsx]: mijn oplossing test beter alle edge cases

File: src/map/components/points-available/PointsAvailable.test.jsx [hunk 2]
```diff
@@ -2,34 +2,32 @@ import React from 'react';
...
-import drawToolConfig from '../../services/draw-tool/draw-tool-config';
 
-const maxMarkers = drawToolConfig.MAX_MARKERS;
-const markersLeftWarning = drawToolConfig.MARKERS_LEFT_WARNING;
-const mockMarkers = [];
+describe('PointsAvailable', () => {
+  it('should render no markers left', () => {
+    const wrapper = shallow(
+      <PointsAvailable
+        markersLeft={0}
```
Comment (jasperswart) [inline src/map/components/points-available/PointsAvailable.test.jsx]: dit het moet {1} zijn

File: src/map/components/points-available/PointsAvailable.test.jsx [hunk 3]
```diff
@@ -2,34 +2,32 @@ import React from 'react';
...
-            />
-          );
-          expect(wrapper).toMatchSnapshot();
-        });
-      });
-    });
+  it('should render multiple markers left', () => {
+    const wrapper = shallow(
+      <PointsAvailable
+        markersLeft={0}
```
Comment (jasperswart) [inline src/map/components/points-available/PointsAvailable.test.jsx]: dit moet {3} zijn oid

File: src/map/services/draw-tool/draw-tool-toggle.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import {
```
Comment (jasperswart) [inline src/map/services/draw-tool/draw-tool-toggle.js]: ja beter, maar mag deze in een eigen directory ?

File: src/map/components/points-available/PointsAvailable.test.jsx (hunk 1 above)
Comment (RRMoelker) [reply src/map/components/points-available/PointsAvailable.test.jsx]: Ja inderdaad. Dit component is wel helemaal getest nu denk ik. Maar ik zal de draw tool nog wat verder met tests die ik uit de PointsAvailable.test.jsx weg heb gehaald.

File: src/map/components/points-available/PointsAvailable.test.jsx (hunk 2 above)
Comment (RRMoelker) [reply src/map/components/points-available/PointsAvailable.test.jsx]: oef, goede

File: src/map/services/draw-tool/draw-tool-toggle.js (hunk 4 above)
Comment (RRMoelker) [reply src/map/services/draw-tool/draw-tool-toggle.js]: Deze hoort eigenlijk bij de draw-tool service. Maar ik kreeg de boel niet gemocked als het in dezelfde file stond. Beetje stomme reden, maar bij het aanpakken van die draw-tool service veranderd er toch nog een boel
